lib/Runtime/Library/ArgumentsObject.h

```diff
@@ -32,6 +30,11 @@ namespace Js
         virtual BOOL AdvanceWalkerToArgsFrame(JavascriptStackWalker *walker) = 0;
     };
 
+    template <> inline bool VarIs<ArgumentsObject>(RecyclableObject* obj)
```

Comment:
Looks like you've changed the semantics a little here, from `Var aValue` to `RecyclableObject* aValue`; is that intentional?

Reply:
Ah, looks like the `Var` case is handled polymorphically by checking it isn't a tagged value, and then passing to the `RecyclableObject` case anyway.